WebDriverAgentLib/Utilities/FBConfiguration.m

```diff
@@ -22,6 +23,7 @@
 static BOOL FBShouldUseCompactResponses = YES;
 static NSString *FBElementResponseAttributes = @"type,label";
 static NSUInteger FBMaxTypingFrequency = 60;
+static BOOL FBShouldLoadSnapshotWithAttributes = NO;
```

Comment:
why not YES?

Reply:
actually, I'd rather assign this to `[XCElementSnapshot.class respondsToSelector:@selector(snapshotAttributesForElementSnapshotKeyPaths:)]` result, since we'd like to have the most optimal approach available by default

Reply:
The thought behind setting this to `NO` as default value was that the current version of getting page source and this works for most devices. But I'm also fine with setting this to `YES` because in the end we only use methods of XCTest and don't do anything on our own. Also, if something goes wrong we fall back to the current way of creating the page source.